### PR TITLE
Make sure that ifcfg-XXXX files work properly

### PR DESCRIPTION
**Pull request by gmkurtzer**
A offering from Bill Comisky in Slack to ensure that nodes boot network predictably for EL8 varients.

### CODE REVIEW

File: containers/Singularity/rocky-8.def [hunk 1]
```diff
@@ -14,4 +14,11 @@ Include: dnf
...
     gzip which util-linux openssh-clients openssh-server dhclient \
     pciutils vim-minimal shadow-utils strace cronie crontabs cpio \
-    wget rocky-release ipmitool yum NetworkManager
\ No newline at end of file
+    wget rocky-release ipmitool yum NetworkManager network-scripts
+
+    systemctl disable NetworkManager
+    systemctl disable NetworkManager-dispatcher
+    systemctl disable NetworkManager-wait-online
+    systemctl enable network-scripts
```
Comment (bcomisky) [inline containers/Singularity/rocky-8.def:22]: oops, I mistyped this originally, should be the following:
`systemctl enable network`

Not a native systemd service, this creates SysV style init script links.

File: containers/Singularity/rocky-8.def [hunk 2]
```diff
@@ -14,4 +14,11 @@ Include: dnf
     shadow-utils rsyslog tzdata util-linux words zlib tar less \
     gzip which util-linux openssh-clients openssh-server dhclient \
     pciutils vim-minimal shadow-utils strace cronie crontabs cpio \
-    wget rocky-release ipmitool yum NetworkManager
\ No newline at end of file
+    wget rocky-release ipmitool yum NetworkManager network-scripts
```
Comment (bcomisky) [inline containers/Singularity/rocky-8.def:17]: Can we just leave off NetworkManager?  If so we can skip the disables, below too.

Comment (gmkurtzer) [reply containers/Singularity/rocky-8.def:17]: Yep, good thinking. I'll do that.

Thanks!

**Comment by gmkurtzer** [comment]
@bcomisky How does that look?

File: containers/Singularity/rocky-8.def (hunk 2 above)
Comment (bcomisky) [reply containers/Singularity/rocky-8.def]: Had some issues this morning with inconsistent eth0/eth1 device naming using this setup.  See warewulf slack thread.. I ended up adding back NetworkManager and excluding all devices except warewulf manged ones.. and still managed to have issues until I changed names from eth0 to ww0.   So not sure if I would've ended up in the same place using network-scripts.  Just not sure it's the way to go.

**Pull request merged**